src/index.html

```diff
@@ -11,7 +11,234 @@
+        </legend>
+        <div class="personal">
+          <label>
+            Surname:
+            <input
+              class="form-surname"
+              type="text"
+              name="surname"
+            >
+          </label>
+        </div>
+        <div class="personal">
+          <label>
+            Name:
+            <input
+              class="form-name"
+              type="text"
+              name="name"
+            >
+          </label>
+        </div>
+
+
+      </fieldset>
+    </form> -->
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <div>
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="surname"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="name"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              required
+              min="1"
+              max="100"
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            Full date of birth:
+            <input
+              type="date"
+              name="date"
+              required
+            >
+          </label>
+        </div>
+
+        <div class="form personal">
+          <label>
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              name="term"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+        <div>
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="E-mail"
+              placeholder="email@example.com"
+              autocomplete="off"
+              required
+            >
+          </label>
+        </div>
+
+        <div>
+          <label>
+            Password:
+            <input
+              class="password"
+              type="password"
+              name="password"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="y"
```

Comment:
```suggestion
              value="yes"
```

better use more meaningful names for attributes